### ISSUE TITLE
‘Reveal all’ doesn't always work

### ISSUE THREAD

**Issue by Antagony1060** [viresh-ratnakar/exolve#23]
I noticed this on a larger (10 x 8) puzzle I'm working on, but I've experimented with smaller sizes and this 4 x 3 grid behaves similarly:
```
exolve-begin
  exolve-width: 4
  exolve-height: 3
  exolve-grid:
    C R E W
    A|N|L|E
    R A I D
exolve-end
```
As you will see, it doesn't fill the last column on the right for some reason. With the larger puzzle it only fills the top row, which is `T H I S|L I N E` and nothing else below it. The 'Reveal this' button seems to work fine on any clue. 

I think it may be related, but I've also noticed with this grid that tab navigation doesn't work at all either.

**Comment by viresh-ratnakar** [comment]
Will debug soon, thanks! On the face of it, it seems that I've accidentally
used width in place of height (or vice versa) somewhere in the code,
creating a bug only exposed by non-square grids.

**Comment by Antagony1060** [comment]
Thanks! I'm sure you're right because taking the bottom two rows off the larger grid makes everything reveal as normal and taking a third row off breaks it again but with a missing column instead of rows.

**Comment by viresh-ratnakar** [comment]
yeah it's a single "gridHeight" instead of "gridWidth" in the revealAll()
function. Will release the fix later tonight.

the tab navigation works once you provide the clues (my tab navigation code
takes its ordering from the list of clues -- I suppose I can and will also
make that work even when no clues are provided, just for completeness)

**Comment by Antagony1060** [comment]
Of course, I should have realised it was the lack of clue lists. With all the other navigation methods still working I guess I just assumed shifting would too.

Please don't rush to publish a fix on my account. I'm not in any hurry, I just found the bug and wanted to let you know about it.

**Comment by viresh-ratnakar** [comment]
I've uploaded the fix in v0.50

**Comment by Antagony1060** [comment]
Tested and working fine now, thanks!
